Objects/longobject.c

```diff
@@ -540,8 +507,8 @@ PyLong_AsLongAndOverflow(PyObject *vv, int *overflow)
             }
         }
         /* Haven't lost any bits, but casting to long requires extra
-         * care (see comment above).
-         */
+            * care (see comment above).
+            */
```

Comment:
Accidental reformat?
```suggestion
         * care (see comment above).
         */
```

Reply:
Fixed.